include/inexor/vulkan-renderer/availability_checks.hpp

```diff
@@ -40,34 +32,30 @@ class AvailabilityChecksManager {
     ~AvailabilityChecksManager() = default;
 
     /// @brief Checks if a certain Vulkan instance layer is available on the system.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkEnumerateInstanceLayerProperties.html
     /// @param instance_layer_name The name of the Vulkan instance layer.
-    /// @return true if the Vulkan instance layer is available, false otherwise.
+    /// @return 'true' if the Vulkan instance layer is available, 'false' otherwise.
     /// @note Available instance layers can be enabled by passing them as parameter during Vulkan instance creation.
     [[nodiscard]] bool has_instance_layer(const std::string &instance_layer_name);
 
     /// @brief Checks if a certain Vulkan instance extension is available on the system.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkEnumerateInstanceExtensionProperties.html
     /// @param instance_extension_name The name of the Vulkan instance extension.
-    /// @return true if the Vulkan instance extension is available, false otherwise.
+    /// @return 'true' if the Vulkan instance extension is available, 'false' otherwise.
     /// @note Available instance extensions can be enabled by passing them as parameter during Vulkan instance creation.
     [[nodiscard]] bool has_instance_extension(const std::string &instance_extension_name);
 
     /// @brief Checks if a certain Vulkan device layer is available on the system.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkEnumerateDeviceLayerProperties.html
     /// @note Device layers and device extensions are coupled to a certain graphics card which needs to be specified as
     /// parameter.
     /// @param graphics_card The selected graphics card.
     /// @param device_layer_name The name of the Vulkan device layer.
-    /// @return true if the Vulkan device layer is available, false otherwise.
+    /// @return 'true' if the Vulkan device layer is available, 'false' otherwise.
     /// @note Available device layers can be enabled by passing them as a parameter during Vulkan device creation.
     [[nodiscard]] bool has_device_layer(const VkPhysicalDevice &graphics_card, const std::string &device_layer_name);
 
     /// @brief Checks if a certain Vulkan device extension is available on the system.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkEnumerateDeviceExtensionProperties.html
-    /// @param graphics_card [in] The selected graphics card.
-    /// @param device_extension_name [in] The name of the Vulkan device extension.
-    /// @return true if the Vulkan device extension is available, false otherwise.
+    /// @param graphics_card The selected graphics card.
+    /// @param device_extension_name The name of the Vulkan device extension.
+    /// @return 'true' if the Vulkan device extension is available, 'false' otherwise.
```

Comment:
```suggestion
    /// @return ``true`` if the Vulkan device extension is available, ``false`` otherwise.
```

Reply:
Done. I removed the `false otherwise` everywhere.